src/libtorch.cc

```diff
@@ -764,7 +799,8 @@ ModelInstanceState::ValidateInputs(const size_t expected_input_cnt)
   // configuration specifies only those.
   std::vector<std::string> allowed_inputs;
 
-  const torch::jit::Method& method = torch_model_->get_method("forward");
+  const torch::jit::Method& method =
```

Comment:
Add error checking here for case the `model_state_->ModuleMethodName()` is not available in the model. 
Otherwise the model loading would fail.

Reply:
@iceychris  Did you get a chance to add the exception handling? I can take over if you don't mind?

Reply:
Not yet, I'm currently busy, so feel free to go ahead 👍🏾